scripts/ci/run_ci.sh

```diff
@@ -16,6 +16,9 @@
 #
 # The script can be run locally using for exmaple:
 # ./scripts/ci/run_ci.sh -b master -r upstream  -p
+
+set -v
```

Comment:
Ok.. `set -v` is just annoying printing all functions and the options switch.. sorry for that.
`set -x` seems to be much more useful.
Note that zephyr-env.sh also gets verbose. If that's what you wanted, then its fine, but if not, you may want to move the set -x just below the call to `source zephyr-env.sh`.